### ISSUE TITLE
Thunderbird 60.8.0 login issues with DavMail 5.3.1

### ISSUE THREAD

**Issue by dejan2609** [mguessan/davmail#55]
Steps to reproduce:
* step 1: use previous environment that works:
  * Thunderbird 60.8.0 (64-bit) / Windows 10 64 bit version 1903 / Oracle Java 1.8.0_201
  * note: Thunderbird has all required passwords saved
  * DavMail 5.2.0 is using EWS exchange protocol (see attached snapshot)
* step 2: stop Thunderbird, uninstall DavMail 5.2.0, install DavMail 5.3.1, start Thunderbird
  * symptoms: Thunderbird asks for passwords, saves them, but can't connect to exchange server

![image](https://user-images.githubusercontent.com/19467899/63083125-350b6300-bf48-11e9-997e-7ffe6103ab36.png)

**Comment by dejan2609** [comment]
More snapshots: Thunderbird with DavMail 5.2.0 (Tools > Options > Security > Saved Passwords): 

![image](https://user-images.githubusercontent.com/19467899/63085032-d1cfff80-bf4c-11e9-9eb8-15295c9193fc.png)

**Comment by dejan2609** [comment]
Thunderbird with DavMail 5.3.1 (fails to authenticate):

![image](https://user-images.githubusercontent.com/19467899/63085150-21aec680-bf4d-11e9-9e39-06fdd86dd0e0.png)

![image](https://user-images.githubusercontent.com/19467899/63085180-355a2d00-bf4d-11e9-94f6-19453d984bec.png)

**Comment by dejan2609** [comment]
**DavMail properties file** (for both DavMail 5.2.0 and 5.3.1): [davmail.properties.txt](https://github.com/mguessan/davmail/files/3504976/davmail.properties.txt)

**DavMail log** (for 5.3.1 version only): [davmail.log](https://github.com/mguessan/davmail/files/3504971/davmail.log)

**Comment by dejan2609** [comment]
Note: this seems related with #52 **_Authentication fails with v5.3.0_**

**Comment by dejan2609** [comment]
@mguessan let me know do you need any more details about this.

**Comment by mguessan** [comment]
Some users have trouble with:
- on premise Exchange
- EWS mode
- form authentication (/owa path) succeeds
- first call to EWS fails (authentication fails)
=> after troubleshooting with another user, it seems that EWS endpoint needs the domain name (domain\user) to authenticate and doesn't receive it with 5.3.1
The issue is that I don't understand how this can work with 5.2.0...

Anyway, quickest workarounds are:
- use direct EWS url => must end with /EWS/Exchange.asmx
- set default domain in DavMail settings or use UPN to authenticate (userprincipalname usually your email but not always, depends on Active Directory configuration)

In order to help me fix this, I would need a WIRE DEBUG log of a successful 5.2.0 authentication.
Please don't send it publicly, use my private email address mguessan@free.fr

**Comment by dejan2609** [comment]
@mguessan well, after few re-installs and moving back an forth between 5.2.0 and 5.3.1 (in order to create WIRE DEBUG log) I managed to authenticate with DavMail 5.3.1 ✔️ 

Details (maybe this could help to someone):
* Exchange protocol was/is **EWS** (for both 5.2.0 and 5.3.1)
* URL is changed:
   * from `https://some.company.com/owa` (in 5.2.0)  
   * to `https://some.company.com/ews/exchange.asmx` (in 5.3.1)

**Comment by mguessan** [comment]
Thanks for your feedback. I am still interested to know why form authentication (/owa) with simple username is working with 5.2.0 and not with 5.3.1.

=> at some point you should have a POST with username and password to owa endpoint, does it contain a domain name ? If yes can we track where it comes from ?

**Comment by dejan2609** [comment]
@mguessan to answer your question: 
 * I use DavMail/Thunderbird to in a domain
 * not 100% sure how Thunderbird saves username (and passes to both 5.2.0 and/or 5.3.1 DavMail), basically there are two possibilities:
    * `username` 
    * `DOMAINNAME\username`

I am in the middle of something, but I will make a note for my self to test this in a week or so with both 5.2.0 and 5.3.1 (it will be done by-the-book, I will remove davmail.properties, logs and passwords saved by a Thunderbird prior to testing). After testing I will post results here (and I will send you WIRE DEBUG log via email).

**Comment by braisarias** [comment]
I have the same problem with:
Thunderbird 60.8.0 (64-bit) from Manjaro oficial repository
DavMail released version: 5.3.1-3079 from AUR repository

I've tried changing my settings, only the following parameters:
```
#it is deprecated in davmail 5.3.1
#davmail.enableEws=true
#use this instead
davmail.mode=EWS
```

and
```
# it seems that this URL fails
#davmail.url=https\://myWebMailServer/owa
#use this instead
davmail.url=https\://myWebMailServer/ews/exchange.asmx
```

I've attached the log file:
[davmail.log](https://github.com/mguessan/davmail/files/3525873/davmail.log)
